lms/urls.py

```diff
@@ -143,7 +143,7 @@
 
 # TODO: This needs to move to a separate urls.py once the student_account and
 # student views below find a home together
-if settings.FEATURES['ENABLE_COMBINED_LOGIN_REGISTRATION']:
+if settings.FEATURES.get('ENABLE_COMBINED_LOGIN_REGISTRATION'):
```

Comment:
Sorry, I had to do these changes to get tests to run o_0 It should probably be this way anyway.